molgenis-data/src/main/java/org/molgenis/data/decorator/DynamicRepositoryDecoratorRegistry.java

```diff
@@ -0,0 +1,15 @@
+package org.molgenis.data.decorator;
+
+import org.molgenis.data.Entity;
+import org.molgenis.data.Repository;
+
+import java.util.stream.Stream;
+
+public interface DynamicRepositoryDecoratorRegistry
+{
+	void addFactory(DynamicRepositoryDecoratorFactory factory);
```

Comment:
Does ```dynamic``` imply that you should also be able to remove a factory?

Reply:
Maybe the words are in the wrong order here 😄 . The decorators are dynamic, the factories aren't.

Reply:
:-D